Lib/test/test_syntax.py

```diff
@@ -972,6 +972,11 @@ def func2():
 """
         self._check_error(code, "invalid syntax")
 
+    def test_invalid_line_continuation_left_recursive(self):
+        # Check bpo-42218: SyntaxErrors following left-recursive rules
+        # (t_primary_raw in this case) need to be tested explicitly
+        self._check_error('A.\u018a\\ ', "unexpected character after line continuation character")
```

Comment:
Btw, have you seen this comment: https://github.com/python/cpython/pull/23058/files#r515498221 ?

Reply:
Addressed it.